src/amo/containers/App.js

```diff
@@ -1,15 +1,35 @@
+/* global window */
+
 import React, { PropTypes } from 'react';
 import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
 
 import 'core/fonts/fira.scss';
 import 'amo/css/App.scss';
 import translate from 'core/i18n/translate';
-
+import { startLoginUrl } from 'core/api';
 
 export class AppBase extends React.Component {
   static propTypes = {
     children: PropTypes.node,
+    handleLogIn: PropTypes.func.isRequired,
     i18n: PropTypes.object.isRequired,
+    isAuthenticated: PropTypes.bool,
+    location: PropTypes.object.isRequired,
+  }
+
+  logIn() {
+    const { handleLogIn, i18n, isAuthenticated, location } = this.props;
+    if (isAuthenticated) {
+      return <p>You are logged in</p>;
```

Comment:
I left this out of the translations so we don't generate any extra work for localisers. Once we have a proper mock for how this should look we will probably link to an account page or show the username instead.